daml-assistant/src/DAML/Assistant/Env.hs

```diff
@@ -146,11 +149,10 @@ getLatestInstalledSdkVersion (DamlPath path) = do
             Left _ -> pure Nothing
             Right dirlist -> do
                 subdirs <- filterM (doesDirectoryExist . (dpath </>)) dirlist
-                -- TODO (FAFM): warn if subdirs /= dirlist
-                --  (i.e. $DAML_HOME/sdk is polluted with non-dirs).
-                let versions = filter ("nightly-" `isPrefixOf`) subdirs
-                    -- TODO (FAFM): configurable channels
-                pure $ fmap (SdkVersion . pack) (maximumMay versions)
+                let preversions = mapMaybe (eitherToMaybe . V.fromText . pack) subdirs
```

Comment:
Should we throw an for subdirs that cannot be parsed instead of silently filtering them?
What does `preversions` stand for? It sounds similar to `prereleases` but that doesn’t seem to be accurate?

Reply:
it's just a variable name -_- I guess `semvers` would be more accurate